src/components/DataTable/DataTable.scss

```diff
@@ -151,7 +151,7 @@ $breakpoint: 768px;
 
 .Cell-total {
   @include text-emphasis-strong;
-  background: color('sky', 'lighter');
+  background: var(--p-surface-foreground, color('sky', 'lighter'));
```

Comment:
Since these are `sky, lighter`, not `white`, I think they should be surface-foreground-subdued. Going to add a change that cleans up subdued a bit so that looks truer to its intent.